lib/msal-browser/src/interaction_client/BaseInteractionClient.ts

```diff
@@ -163,10 +163,10 @@ export abstract class BaseInteractionClient {
 
         if (requestAuthority) {
             this.logger.verbose("Creating discovered authority with request authority");
-            return await AuthorityFactory.createDiscoveredInstance(requestAuthority, this.config.system.networkClient, this.browserStorage, authorityOptions);
+            return await AuthorityFactory.createDiscoveredInstance(requestAuthority, this.config.system.networkClient, this.browserStorage, authorityOptions, undefined);
```

Comment:
Why is this `undefined` now needed here?

Reply:
Not needed, addressed.